pandas/core/frame.py

```diff
@@ -11264,6 +11264,15 @@ def idxmin(
         indices = res._values
         # indices will always be np.ndarray since axis is not N
 
+        if (indices == -1).any():
```

Comment:
Based on the error warning message, shouldn't this be `(indices == -1).all() or ((indices == -1).any() and not skipna)`?

Reply:
no.  that check is in the `mask.all() or (mask.any() and not skipna)` check that is inside nanargminmax that this calls.